Python/Product/Analysis/Analysis.csproj

```diff
@@ -54,6 +54,10 @@
     <Reference Include="System.Data.DataSetExtensions" />
     <Reference Include="System.Data" />
     <Reference Include="System.Xml" />
+    <Reference Include="Newtonsoft.Json">
+      <HintPath>$(PackagesPath)\Newtonsoft.Json\lib\net45\Newtonsoft.Json.dll</HintPath>
```

Comment:
I believe adding `<Reference Include="Newtonsoft.Json" />` should be enough, the rest should be handled by build script.
@zooba , can you please take a look?

Reply:
It is not really needed atm, I will remove it. This was for Pythia experiment.